lib/uiautomator2.js

```diff
@@ -265,6 +265,8 @@ class UiAutomator2Server {
     if (_.isBoolean(this.disableSuppressAccessibilityService)) {
       cmd.push('-e', 'DISABLE_SUPPRESS_ACCESSIBILITY_SERVICES', this.disableSuppressAccessibilityService);
     }
+    // Disable Google analytics to prevent possible fatal exception
+    cmd.push('-e', 'disableAnalytics', true);
```

Comment:
I assume `true` should be a string

Reply:
link: https://github.com/appium/appium-espresso-driver/pull/666#discussion_r614686105